lightningd/peer_htlcs.c

```diff
@@ -500,22 +500,28 @@ static void forward_htlc(struct htlc_in *hin,
 	struct amount_msat fee;
 	struct lightningd *ld = hin->key.channel->peer->ld;
 	struct channel *next = active_channel_by_id(ld, next_hop, NULL);
+	u32 fee_base = ld->config.fee_base;
+	u32 fee_ppm = ld->config.fee_per_satoshi;
 
 	/* Unknown peer, or peer not ready. */
 	if (!next || !next->scid) {
 		local_fail_htlc(hin, WIRE_UNKNOWN_NEXT_PEER, NULL);
 		return;
 	}
 
+	/* Fee base and ppm can also be set per channel */
+	if (next->feerate_base > 0)
+		fee_base = next->feerate_base;
+	if (next->feerate_ppm > 0)
+		fee_ppm = next->feerate_ppm;
+
```

Comment:
I would always initialize it, and node do this here.

Reply:
fixed